database/src/main/kotlin/no/nav/su/se/bakover/database/behandlinger/stopp/StoppbehandlingJdbcRepo.kt

```diff
@@ -0,0 +1,99 @@
+package no.nav.su.se.bakover.database.behandlinger.stopp
+
+import kotliquery.Row
+import kotliquery.sessionOf
+import kotliquery.using
+import no.nav.su.se.bakover.database.ObjectRepo
+import no.nav.su.se.bakover.database.hentListe
+import no.nav.su.se.bakover.database.oppdatering
+import no.nav.su.se.bakover.database.uuid
+import no.nav.su.se.bakover.database.uuid30
+import no.nav.su.se.bakover.domain.Attestant
+import no.nav.su.se.bakover.domain.Saksbehandler
+import no.nav.su.se.bakover.domain.behandlinger.stopp.Stoppbehandling
+import no.nav.su.se.bakover.domain.behandlinger.stopp.StoppbehandlingRepo
+import java.util.UUID
+import javax.sql.DataSource
+
+class StoppbehandlingJdbcRepo(
+    private val dataSource: DataSource,
+    private val objectRepo: ObjectRepo
+) : StoppbehandlingRepo {
+
+    override fun opprettStoppbehandling(nyBehandling: Stoppbehandling.Simulert) =
+        using(sessionOf(dataSource)) { session ->
+            "insert into stoppbehandling (id, opprettet, sakId, status, utbetaling, stoppÅrsak, saksbehandler) values (:id, :opprettet, :sakId, :status, :utbetaling, :stoppArsak, :saksbehandler)".oppdatering(
+                params = mapOf(
+                    "id" to nyBehandling.id,
+                    "opprettet" to nyBehandling.opprettet,
+                    "sakId" to nyBehandling.sakId,
+                    "status" to nyBehandling.status,
+                    "utbetaling" to nyBehandling.utbetaling.id.toString(),
+                    "stoppArsak" to nyBehandling.stoppÅrsak,
+                    "saksbehandler" to nyBehandling.saksbehandler.id,
+                ),
+                session = session
+            ).let {
+                nyBehandling
+            }
+        }
+
+    override fun hentPågåendeStoppbehandling(sakId: UUID): Stoppbehandling? {
```

Comment:
Mulig vi burde tenke litt rundt samtidige behandlinger sånn generelt også. Kan hende vi kunne vært tjent med å bare tillate 1 åpen behandling (uavhengig av type) pr sak til å begynne med.

Reply:
Jeg er åpen for det, men høres ut som et fag-valg. Vi kan fronte det på standup.